src/api/db/migrate/20170607105625_use_rails_enums_binary_releases_table.rb

```diff
@@ -0,0 +1,37 @@
+class UseRailsEnumsBinaryReleasesTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table(:binary_releases) do |t|
+      t.column :new_operation, :integer, limit: 2, default: 0
+    end
+
+    BinaryRelease.operations.values.each do |index|
+      # We need to use raw sql to avoid conflicts with the ActiveRecord::Enum
+      # mapping defined in the model.
+      # SQL enums start at 1, ActiveRecord::Enum usually at 0.
+      BinaryRelease.connection.
+        execute("UPDATE binary_releases SET new_operation='#{index}' WHERE operation='#{BinaryRelease.operations.key(index)}'")
+    end
+
+    remove_column :binary_releases, :operation
+    rename_column :binary_releases, :new_operation, :operation
+  end
```

Comment:
please please triple check this with all our production instances. This is really sensitive data and we can't reimport from backend like most other data in case of error. (if the operations.value order is wrong by bad luck the data will be corrupted).